app/src/main/res/values/strings.xml

```diff
@@ -482,8 +482,7 @@ Scanning will happen automatically.</string>
   <string name="code_sent">Code sent!</string>
   <string name="confirm_your_identity">Confirm your identity to continue.</string>
   <string name="export_vault_warning">This export contains your vault data in an unencrypted format. You should not store or send the exported file over unsecure channels (such as email). Delete it immediately after you are done using it.</string>
-  <string name="enc_export_key_warning">This export encrypts your data using your account\'s encryption key. If you ever rotate your account\'s encryption key you should export again since you will not be able to decrypt this export file.</string>
-  <string name="enc_export_account_warning">Account encryption keys are unique to each Bitwarden user account, so you can\'t import an encrypted export into a different account.</string>
+  <string name="export_vault_file_pw_protect_info">This file export will be password protected and require the file password to decrypt.</string>
```

Comment:
really hoping there is some automation around string resources?

Reply:
There is. We use Crowdin. When this PR is merged the new string will be uploaded. The old string and its translations will be deleted. Translations will be automatically synced on our regular schedule as they're provided.